peakina/readers/excel.py

```diff
@@ -0,0 +1,133 @@
+"""
+Module to add xml support
+"""
+import enum
+from io import StringIO
+from typing import Any, List, Optional
+
+import pandas as pd
+import xlrd
+from openpyxl import load_workbook
+from openpyxl.utils.exceptions import InvalidFileException
+
+from peakina.readers.csv import PreviewArgs
+
+
+class EXCEL_TYPE(enum.Enum):
+    NEW = "new"
+    OLD = "old"
+
+
+def _read_old_xls_format(wb: Any, sh_name: str, preview: Optional[PreviewArgs]) -> Any:
+    """ """
```

Comment:
Feel free to either remove the docstring or add a clear one

Reply:
👍🏾 Updated !